packages/liveblocks-react-lexical/src/comments/anchored-threads.tsx

```diff
@@ -313,17 +314,19 @@ function ThreadWrapper({
     });
   }
 
-  const handleRef = useCallback(
-    (el: HTMLDivElement) => {
-      onItemAdd(thread.id, el);
-      return () => onItemRemove(thread.id);
-    },
-    [thread.id, onItemAdd, onItemRemove]
-  );
+  useEffect(() => {
```

Comment:
We could use `useLayoutEffect` here instead since that would be closer to a ref callback in the lifecycle

Reply:
Good idea! Also added a CHANGELOG like @nvie suggested.